lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -302,6 +309,19 @@ private JourneyResponse getJourneyResponse(
         }
     }
 
+    @Tracing
+    private List<VerifiableCredential> getVerifiableCredentials(
+            String userId, String evcsAccessToken)
+            throws CredentialParseException, EvcsServiceException {
+        if (configService.enabled(EVCS_READ_ENABLED)) {
+            var vcs = evcsService.getVerifiableCredentials(userId, evcsAccessToken, CURRENT);
```

Comment:
This is ok for now, but one thing we'll want to do (as mentioned on @rpayal's PR) is to avoid reading twice from EVCS (here and in the migration process) - we don't need to do a migration at all if the identity is already in EVCS.

It'll probably be easier to do this as part of the F2F journey work, since then we'll be reading CURRENT + PENDING, and need to handle the pending case separately.